src/schemes/fluid/fluid.jl

```diff
@@ -184,11 +185,11 @@ function system_data(system::FluidSystem, v_ode, u_ode, semi)
     density = current_density(v, system)
     pressure = current_pressure(v, system)
 
-    return (; coordinates, velocity, mass, density, pressure)
+    return (; coordinates, velocity, mass, density, pressure, acceleration=dv)
```

Comment:
`dv` is not the acceleration.
But I think it should work to apply `current_velocity(dv, system)` to get the acceleration.